frontend/src/components/StreamingBox.vue

```diff
@@ -147,28 +162,26 @@ export default {
 
   data() {
     return {
-      stickyStreamingSrc: null,
-      isVideoAvailable: false,
-      isVideoVisible: false,
-      remainingSecondsCurrentVideoCycle: 30,
-      remainingSecondsUntilNextCycle: -1,
-      currentBitrate: null,
+      STREAM_STATUS,
+      streamStatus: STREAM_STATUS.Unavailable,
+      isTrackMuted: false,
+      isVideoLoading: false,
+
       slowLinkLoss: 0,
-      slowLinkShowing: false, // show on mousenter
-      slowLinkHiding: false, // hide on moseleave
-      trackMuted: false,
-      videoLoading: false,
-      printerStockImgSrc: '#svg-3d-printer'
+      slowLinkLossLimit: 50,
+      isSlowLinkShowing: false,  // show on mousenter
+      isSlowLinkHiding: false,  // hide on moseleave
+      currentBitrate: null,
+
+      remainingSeconds: StreamThrottle.defaultRemainingSeconds(),
+      activeSessionRemainingSeconds: StreamThrottle.calculateSessionsSeconds()[0],
+      frozenSessionRemainingSeconds: StreamThrottle.calculateSessionsSeconds()[1],
+      basicStreamTimerId: null,
+      currentSession: STREAM_SESSION.Idle,
```

Comment:
Tried to encapsulate basic streaming timer logic with all the math as much as possible